distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -863,6 +869,9 @@ public void testUpdateContentResourceUri() throws IOException {
 
     @Test
     public void testCachedContentLengthHeader() throws IOException {
+
+        console.runCommand(CLEAR_CACHE);
```

Comment:
Is there any type of wait we can do to assure the cache has been cleared before continuing the test?